src/components/SMARTBox/PatientBox.js

```diff
@@ -316,11 +318,32 @@ export default class PatientBox extends Component {
       })
       .then(result => {
         this.setState({ questionnaireResponses: result });
+      })
+      .then(() => this.getQuestionnaireTitles());
+  }
+
+  getQuestionnaireTitles() {
+    const client = FHIR.client(this.props.params);
```

Comment:
The smart pr added a client to the props, we can use that instead.

Reply:
which branch?

Reply:
Ah I see, origin/dev was updated with the PR- I'll rebase my branch off of it